src/topology/homeomorph.lean

```diff
@@ -435,6 +435,22 @@ def image (e : α ≃ₜ β) (s : set α) : s ≃ₜ e '' s :=
 
 end homeomorph
 
+/-- An equiv between a space `α` and a topological space `β` becomes a homeomorphism when `α`
+is endowed with the induced topology. -/
+@[simps] def equiv.to_homeomorph {α β : Type*} [t : topological_space β] (f : α ≃ β) :
```

Comment:
There are two natural candidates for this name: `induced` topology on `α` and `coinduced` toopology on `β`. Also, I think that it's better to use `[topological_space α] (hf : inducing f)`.

Reply:
Done with inducing.